coral/src/domain/topic/topic-api.ts

```diff
@@ -23,34 +20,26 @@ import {
   KlawApiRequestQueryParameters,
   KlawApiResponse,
 } from "types/utils";
+import { convertQuery } from "src/services/api-helper";
 
-const getTopics = async ({
-  currentPage = 1,
-  environment = "ALL",
-  teamName,
-  searchTerm,
-}: {
-  currentPage: number;
-  environment: string;
-  teamName: Team;
-  searchTerm?: string;
-}): Promise<TopicApiResponse> => {
+const getTopics = async (
+  params: KlawApiRequestQueryParameters<"getTopics">
+): Promise<TopicApiResponse> => {
   // "ALL_TEAMS_VALUE" represents topic list without
   // the optional team parameter
   // where we still need a way to represent an
   // option for "Select all teams" to users
-  const team = teamName !== ALL_TEAMS_VALUE && teamName;
+  // const team = teamName !== ALL_TEAMS_VALUE && teamName;
```

Comment:
Should this be deleted? ^^

Reply:
Yes! 🙈  removed it. Thanks for catching!